src/Aspire.Hosting/Eventing/IDistributedApplicationEvent.cs

```diff
@@ -0,0 +1,8 @@
+namespace Aspire.Hosting.Eventing;
+
+/// <summary>
+/// Represents an event that is published during the lifecycle of the AppHost.
+/// </summary>
+public interface IDistributedApplicationEvent
```

Comment:
Should this have the resource or should there be a derived event that has the resource? (IDistributedApplicationResourceEvent)

Reply:
See latest push.